nnvm/src/top/tensor/transform.cc

```diff
@@ -879,5 +879,48 @@ Examples::
     return Array<Tensor>{ topi::flip(inputs[0], param.axis) };
 });
 
+// CropLike
+DMLC_REGISTER_PARAMETER(CropLikeParam);
+
+inline bool CropLikeShape(const nnvm::NodeAttrs& attrs,
+                          std::vector<TShape>* in_attrs,
+                          std::vector<TShape>* out_attrs) {
+  CHECK_EQ(in_attrs->size(), 2U);
+  CHECK_EQ(out_attrs->size(), 1U);
+  NNVM_ASSIGN_OUTPUT_SHAPE(attrs, *out_attrs, 0, (*in_attrs)[1]);
+  return true;
+}
+
+NNVM_REGISTER_OP(crop_like)
+.describe(R"code(Crop the 2nd and 3rd dim of input data,
+with width and height of the second input symbol.
```

Comment:
A new operator to call tvm/crop directly

Reply:
Issue for a single crop operator in nnvm is that number of inputs can be either 1 or 2. It requires an extra parameter to hint whether it is a "crop_like" parameter. I can add another crop operator in nnvm which just takes 1 input tensor.